x-pack/test/security_solution_api_integration/config/serverless/config.base.ts

```diff
@@ -43,7 +43,7 @@ export function createTestConfig(options: CreateTestConfigOptions) {
 
       mochaOpts: {
         ...svlSharedConfig.get('mochaOpts'),
-        grep: '/^(?!.*@skipInServerless).*@serverless.*/',
+        grep: '/^(?!.*(^|\\s)@skipInServerless(\\s|$)).*@serverless.*/',
```

Comment:
The regex should be revisited since we found out that there are some scenarios that are not excluded and it should like:

`@ess @serverless @skipInServerless check_privileges`

Reply:
We revisited the regex as well as the tests that we were expecting to be included and excluded, and is working fine, was a problem about the use of https://regex101.com/ with the extra `\`